src/main/java/com/marklogic/developer/corb/AbstractManager.java

```diff
@@ -356,15 +363,21 @@ protected String getOption(String[] commandlineArgs, int argIndex, String proper
      * @param propertyName
      * @return the trimmed property value
      */
-    protected String getOption(String argVal, String propertyName) {
-        if (isNotBlank(argVal)) {
-            return argVal.trim();
-        } else if (isNotBlank(System.getProperty(propertyName))) {
-            return System.getProperty(propertyName).trim();
-        } else if (this.properties.containsKey(propertyName) && isNotBlank(this.properties.getProperty(propertyName))) {
-            return this.properties.getProperty(propertyName).trim();
+    protected String getOption(String argVal, String propName) {
+        String retVal=null;
+    	if (isNotBlank(argVal)) {
+    		retVal= argVal.trim();
+        } else if (isNotBlank(System.getProperty(propName))) {
+        	retVal= System.getProperty(propName).trim();
+        } else if (this.properties.containsKey(propName) && isNotBlank(this.properties.getProperty(propName))) {
+        	retVal = this.properties.getProperty(propName).trim();
+            this.properties.remove(propName); //remove from properties file as we would like to keep the properties file simple. 
```

Comment:
This is a problem. I believe that we used to remove the items from properties a while back, but took this line out. We need properties to be retained (currently) for how we manage properties and options, or some classes won't have CORB options available. When Bhagat refactors the options stuff this will get moved and refactored out anyhow.